dio/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ## 5.1.2
 
 - Allow `FormData` to send a null entry value as an empty string.
+- Deprecate `DioError` in favor of `DioException`.
```

Comment:
The entry is placed in the wrong place. It should be under the unreleased section.